src/strategies/lodestar-vesting/index.ts

```diff
@@ -0,0 +1,49 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = '0xAppo';
+export const version = '0.1.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+if (
+      options.beneficiaryAddresses.length >= 25 ||
+      options.contractAddresses.length >= 25
+    ) {
+      console.error('ERROR: Too many stake pool addresses provided.');
+      return;
+    }
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) => {
+    if (options.beneficiaryAddresses.includes(address)) {
+      const index = options.beneficiaryAddresses.indexOf(address);
+      const contractAddress = options.contractAddresses[index];
+      multi.call(address, options.address, 'balanceOf', [contractAddress]);
+    } else {
+      return;
+    }
```

Comment:
```suggestion
    }
```

Reply:
Not sure I understand this comment, the forEach loop is closed by brackets and parentheses in the next line

Reply:
I mean writing a return here will not effect anything right?

Reply:
Done.

Reply:
Done.